sdk/cognitiveservices/azure-ai-inkrecognizer/azure/ai/inkrecognizer/_models.py

```diff
@@ -0,0 +1,722 @@
+#-------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#--------------------------------------------------------------------------
+
+from collections import OrderedDict
+from ._enums import InkRecognitionUnitKind, ShapeKind
+
+
+_STR_MAX_LENGTH = 1024
+
+
+def _truncate(string):
+    if len(string) > _STR_MAX_LENGTH:
+        return string[:_STR_MAX_LENGTH] + "..."
+    return string
+
+
+class Point:
```

Comment:
I would use a named tuple. Or at least allow a tuple to be used as input.

Reply:
This is geometric point that indicates points in return values, such as center of a shape or vertices of bounding rectangle. 

For the ink points in ink stroke, it is inside "_ink_stroke.py" as "IInkPoint". And yes, developer can also input namedtuple as ink point, as long as it has x and y. I have sample code to use namedtuple in readme and will be in samples.

Reply:
Code snippet in readme:


```Python
from azure.ai.inkrecognizer import InkStrokeKind

InkPoint = namedtuple("InkPoint", "x y")

class InkStroke():
    def __init__(self,
                 ink_stroke_id,
                 ink_points,
                 stroke_kind=InkStrokeKind.UNKNOWN,
                 stroke_language=""):
        self.id = ink_stroke_id
        self.points = ink_points
        self.kind = stroke_kind
        self.language = stroke_language
```